### ISSUE TITLE
Go Live Checklist

### ISSUE THREAD

**Issue by murny** [ualbertalib/jupiter#224]
# Epic for capturing go-live tasks for all teams:

### Pre-launch blockers:

- [x] Multi-valued fields (#568)
- [x] page titles (Is this already done?)
- [ ] Completed user and object migration and audit (Metadata also reviewing this)
- [x] Google Analytics/Piwik set up 
- [x] make sure [all links on the site](https://gist.github.com/pgwillia/97fb2931254c02057418) is working, contact info is correct
- [x] make sure the collection image links on the homepage are pointing at the correct collections (#584)
- [x] Google Analytics set up - connect to the production instance. 
- [ ] Pushmi-Pullyu deployment
- [x] Robots.txt changed before go live (this is just a [Rails env variable change](https://github.com/ualbertalib/jupiter/blob/master/app/views/robots/robots.text.erb#L4) on Prod)
- [x] Pre-populate Redis download/view counts (Peter working on this)
- [x] DOI merged
- [x] #557
- [x] #519 (static text)
- [x] #556
- [x]  #581
- [x] Only admins can see or use visibility facet
- [x] Created displaying created not sort_year
- [x] #596
- [x] #597 
- [x] Henry to confirm w/ Tyler re: return from IST SAML server pointing at era.library.ualberta.ca instead of era-beta.library.ualberta.ca

### Can happen post-launch 

- [x] Completed accessibility scans and fixes with automated tools (Wave/AxE) #220 
- [x] Keyboard tests #223 
- [x] Fix thumb-nailing issue on production
- [x] meta tags
- [x] set up the 404 page
- [x] responsive testing on various devices
- [x] Make sure we have a favicon in place
- [x] Cross-Browser check - check site in all mainstream browsers: Chrome, Firefox, IE, Safari. 
- [x] Graceful degradation check, try functions without javascript
- [x] Google Search Console set up
- [ ] Other SEO configuration checks
- [x] check against: https://github.com/thedaviddias/Front-End-Checklist (issue found here: #562)
- [x] check against: http://a11yproject.com/checklist.html
- [x] #543 
- [x] #580 
- [x] #468
- [x] OAI data checking complete
- [ ] Finish load of related objects 

## Tests
- [ ] Completed user acceptance test and usability test. 
- [x] Performance testing
  - [x] automated Jmeter testing (Ops Team)
  - [x] stress testing with concurrent users

## Ops

- [ ] Operationalize the application - on-call process
- [x] Monitoring
- [x] https certificates
- [x] skylight issues (still seeing some error messages, may need dev team to look into)
- [x] thumbnails not appearing (generating?)
- [ ] Fedora disaster recover
- [x] Nessus Scan of App server with skylight/rollbar disabled.
## Metadata
- [ ] Audit of all objects in production triplestore complete
- [ ] Deploy final release

## Data Migration
- [ ] Tasks for #248 are all completed. 
- [ ] Convert sort_date in migrated items from string to int (because of #536) ~~when final release is deployed (```bundle exec rake cleanup:reindex``` can do this - confirm)~~ after #572 is merged and deployed, which won't necessarily happen prelaunch
- [ ] Audit the number of objects migrated
- [ ] remove beta testing data (any object that doesn't have a hydra_noid should be removed)
- [ ] Data cleaning (use a complete list of objects in hydra_noid, and triplestore to build graphs for these objects, and discover orphaned triples/objects)
- [ ] Final incremental migration
- [ ] Generate and load final analytics data (requires dump of ERA mysql data)
- [ ] Load broken datastreams (can happen post-launch if necessary)
- [ ] Remove [Temporary Community](https://era.library.ualberta.ca/communities/b50523c6-5037-40a0-b511-10aa6b17fe12): For use in Jupiter migration

## Communication
